### ISSUE TITLE
Podcasts: If the title in the xml contains CDATA stuff, the title is not shown on the frontend

### ISSUE THREAD

**Issue by brio1009** [splitti/MuPiBox#87]
This happens for example with  https://api.sr.se/api/rss/pod/25889 where the title is
```
<title>
<![CDATA[ August Strindberg – Sveriges största författare ]]>
</title>
```
Also see https://stackoverflow.com/questions/2784183/what-does-cdata-in-xml-mean

Discord thread: https://discord.com/channels/879874342203306005/1196509249489088512/1298756434451894343

**Comment by brio1009** [comment]
probably related to #74

**Comment by brio1009** [comment]
closed by #100
